compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2576,7 +2576,13 @@ TR::Register *OMR::Power::TreeEvaluator::vloadEvaluator(TR::Node *node, TR::Code
    switch(node->getDataType())
      {
      case TR::VectorInt8:
+   cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) ? opcode = TR::InstOpCode::lxvb16x : opcode = TR::InstOpCode::lxvw4x;
```

Comment:
Is there a problem with indentation here? As well as below?

Reply:
The indentation for this entire switch statement is kind of wonky, to be honest (same for the vstore one). I'll fix it up